src/templates/next/layouts/Content/Content.stories.tsx

```diff
@@ -21,7 +21,33 @@ Default.args = {
   layout: "content",
   site: {
     siteName: "Isomer Next",
-    siteMap: [],
+    siteMap: {
+      title: "Isomer Next",
+      permalink: "/",
+      children: [
+        {
+          title: "Parent page",
+          permalink: "/parent",
+          children: [
+            {
+              title: "Irrationality",
+              permalink: "/parent/rationality",
+              children: [
+                {
+                  title: "For Individuals",
+                  permalink: "/parent/rationality/child-page-2",
+                },
+                {
+                  title: "Steven Pinker's Rationality",
+                  permalink: "/parent/rationality/child-page-2",
+                },
+              ],
+            },
+            { title: "Sibling", permalink: "/sibling" },
```

Comment:
nit: can add children here to verify that those children cannot be seen. Same for other pages on the parents level (aunt/uncle)